src/foam/nanos/crunch/ui/CapableObjectWizardlet.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.CLASS({
+  package: 'foam.nanos.crunch.ui',
+  name: 'CapableObjectWizardlet',
+  extends: 'foam.u2.wizard.BaseWizardlet',
+
+  imports: [
+    'crunchController'
+  ],
+
+  requires: [
+    'foam.core.Action',
+    'foam.core.Property',
+    'foam.layout.Section',
+    'foam.layout.SectionAxiom'
+  ],
+
+  properties: [
+    // Properties specific to CapabilityWizardSection
+    {
+      name: 'capability'
+    },
+    {
+      name: 'targetObject',
+      class: 'FObjectProperty',
+      of: 'foam.nanos.crunch.Capable'
```

Comment:
```suggestion
      of: 'foam.nanos.crunch.lite.Capable'
```

Reply:
Oops, I didn't even realize that was here. Pushed some deletions to that since some of those methods were just copied from CapabilityWizardlet.